src/engine/game.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+import RenderEngine from './services/render-engine';
+
+export default class Game {
+
+  running
+
+  constructor(canvas) {
+    this.canvas = canvas;
+    this.gameObjects = [];
+    this.running = false;
+    this.sleep = function (ms) {
```

Comment:
Why is this assigned this way and not declared as a class-level function? Also, should we separate this into a service later on so we can get to it from everywhere or do you think this is the only place we will need it?